### ISSUE TITLE
[FR] React example

### ISSUE THREAD

**Issue by serkandurusoy** [jagi/meteor-astronomy-examples#4]
Given all the traction react gets, it would be great to see first-class-citizen react support (react counterparts of the spacebars helpers etc) and accompanying examples.

I'm not yet familiar with react but I'll soon be starting yet another meteor project with which I really want to give react and astronomy a go. I had been waiting for astronomy@1.0.0 for a few months now and I'm excited to try the astronomy+react combo.

**Comment by lukejagodzinski** [comment]
I haven't been playing with React yet but I will probably give it a try. However, it would be nice if someone (maybe you :) ) could create such example and give some feedback. I don't know if it will work with React without any problems, maybe it will. So, help in this matter would be welcome :)

**Comment by serkandurusoy** [comment]
Yeah, why not. I'll be starting off with the new projects in a few weeks and when I do, I'll perhaps spend the first few days trying to create an example to accompany this repo if noone beats me to it :)

**Comment by mbrookes** [comment]
Seconded. I've started a project with Meteor and React and would like to use Astronomy, but having bypassed Blaze I'm having a hard time grokking the example app as the syntax is unfamiliar. I'm loath to learn Blaze when I'm not going to use it, so even the most simple example with React would be very welcome.

**Comment by lukejagodzinski** [comment]
I will be learning React probably next week. I have a few tutorials to watch and after that I will think about making example for React. However @serkandurusoy may be first :)

**Comment by mbrookes** [comment]
Look forward to it. The official Meteor React tutorial is pretty straightforward. I would largely avoid getting bogged down in Flux philosophy at the outset; one way binding (props & state) is all you need to know to get started, especially given Meteor as a reactive store.

**Comment by JesperWe** [comment]
I have successfully converted the standard Meteor React tutorial app to Astronomy, and it's all the better for it, I think. Having React pass around Astro instances seems to work nicely.

https://github.com/JesperWe/simple-todos-react-astro
